### ISSUE TITLE
Vue: Update docs for Vue 3 release

### ISSUE THREAD

**Issue by emccorson** [OnsenUI/OnsenUI#2861]
The docs will need updated for the Vue 3 release.

Add changes needed here as they come up;

- ~UMD users will now need to manually register the components they want to use.~ Abandoning this idea.
- `v-ons-page`'s events (e.g. `show`) will bubble up to `v-ons-tabbar` so it can listen to them. This already was the case for Vue 2 and we were already using it in the examples, but it's not obvious to me that it's expected behaviour, so we should document that it is expected.
-~~`dialogCancel` event needs documented for VOnsDialog, VOnsAlertDialog, and VOnsPopover. See related core issue: #2885.~~ EDIT: not going to do as we are removing `dialogCancel` instead (#2888).
- ~~Remove `modechange` from the VOnsSplitterSide docs. This is automatically generated from the core docs but since we are filtering it out (#2889), we need a way of stopping the filtered events showing up in the Vue docs.~~ EDIT; not going to do. See issue #2889.
- Remove `ons.preload` from the Vue docs since it doesn't make sense for Vue. As far as I understand, plain JS Onsen handles loading pages through XMLHttpRequest, but Vue Onsen doesn't handle the loading of components. See #2371.
- Remove `checked` prop from VOnsSwitch docs as the behaviour is already covered by v-model. This will need to be done after #2901.

**Comment by thomasbalsloev** [comment]
Yes, please update the docs and code examples to fit Vue 3. 
It is worrying that your docs are not up to date. 

Are you still maintaining this?

**Comment by emccorson** [comment]
Thanks for the comment. Yes, we will do that too. We are still maintaining
(but obviously there is quite a lot to fix at the moment!).

>

**Comment by emccorson** [comment]
Closing as all tasks either done or not going to do due to requirements changes.
